languages/PL.json

```diff
@@ -0,0 +1,28 @@
+{
+  "outageStrings": {
+    "upString1": "jest z powrotem online!",
+    "upString2": "ma poprawioną wydajność.",
+    "degradedString1": "doświadcza problemy.",
```

Comment:
This doesn't follow proper conjugation. Imo it could be replaced with `zmaga się z problemami` (is dealing with issues) for a less literal and more context-aware translation.

Reply:
I believe both methods are appropiate.

Reply:
No. "doświadcza problemy" is improper conjugation. If something it should be "doświadcza problemów". Furthermore "doświadcza" is less formal and doesn't fit as much as my suggestion. Think "nasza strona doświadcza obecnie problemów" vs "nasza strona zmaga się obecnie z problemami". The 2nd one sounds better.